sites/fast-component-explorer/webpack.config.js

```diff
@@ -28,24 +28,23 @@ module.exports = (env, args) => {
             filename: "[name]-[contenthash].js",
         },
         optimization: {
-            runtimeChunk: "single",
-            splitChunks: {
-                chunks: "all",
-                maxInitialRequests: 100,
-                cacheGroups: {
-                    vendor: {
-                        test: /[\\/]node_modules[\\/]/,
-                        name: module => {
-                            const packageName = module.context.match(
-                                /[\\/]node_modules[\\/](.*?)([\\/]|$)/
-                            )[1];
-
-                            // npm package names are URL-safe, but some servers don't like @ symbols
-                            return `npm.${packageName.replace("@", "")}`;
-                        },
-                    },
-                },
-            },
+            // runtimeChunk: "single",
```

Comment:
What's the reason to comment out so much and committing these changes?

Reply:
Updated, this webpack now mirrors the `fast-creator` webpack with comments to indicate why certain sections are commented out.